src/component/NavBar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please remove the `eslint-disable` statement from your code. `Eslint` enables us to write clean, readable and industry-standard codes and using an `eslint-disable` defeats this purpose. Kindly remove it. 👍 😎